website/blog/2023-05-01-package-exports-support.md

```diff
@@ -0,0 +1,229 @@
+---
+title: 'Package Exports Support in React Native'
+authors: [huntie]
+tags: [announcement, metro]
+date: 2023-05-01
+---
+
+# Package Exports Support in React Native
+
+With the release of React Native 0.72, Metro — our JavaScript build tool — now includes experimental support for the `package.json` [`"exports"` field](https://nodejs.org/docs/latest-v18.x/api/packages.html#exports). [When enabled](#enabling-package-exports-experimental), it adds the following functionality:
+
+- [React Native projects will work with more npm packages out-of-the-box](#for-app-developers)
+- [New capabilities for packages to define their API and target React Native](#for-package-maintainers-preview)
+- [Some breaking changes to package resolution (in edge cases)](#breaking-changes)
+
+In this post we'll cover how Package Exports works, and what these changes mean for you as a React Native app developer or package maintainer.
+
```

Comment:
```suggestion

<!--truncate-->

```

Small Docusaurus related nit, let's add truncate tag for example here, or somewhere else in the document, so we do not display full post content on the blog posts list.